### PR TITLE
Fix WMTS-based map issues, and reporting as another user

### PR DESCRIPTION
**Pull request by dracos**

### CODE REVIEW

File: templates/web/base/report/new/form_user_loggedin.html [hunk 1]
```diff
@@ -1,20 +1,29 @@
...
+[% ELSE %]
   [% can_contribute_as_another_user = c.user.has_permission_to("contribute_as_another_user", bodies.keys.join(",")) %]
   [% can_contribute_as_body = c.user.from_body AND c.user.has_permission_to("contribute_as_body", bodies.keys.join(",")) %]
-
   [% IF can_contribute_as_another_user OR can_contribute_as_body %]
+    [% INCLUDE form_as %]
+  [% END %]
+[% END %]
+
+[% BLOCK form_as %]
```
Comment (davea) [inline templates/web/base/report/new/form_user_loggedin.html]: This is now being shown to all users when clicking the map on `/around`. It just has a single 'Yourself' option for unprivileged users. It does disappear if you refresh the `/report/new?...` URL.

Comment (dracos) [inline templates/web/base/report/new/form_user_loggedin.html]: Whoops, I meant for the server to not output anything in that case, fix pushed.

Comment (davea) [inline templates/web/base/report/new/form_user_loggedin.html]: Looks good!

**Comment by davea** [comment]
👍

**Comment by dracos** [comment]
Squashed and merged at e51bb2a

**Pull request closed without merging**